apps/vr-tests-react-components/src/stories/Input.stories.tsx

```diff
@@ -41,6 +41,15 @@ storiesOf('Input Converged', module)
     </div>
   ))
   .addStory('Disabled', () => <Input disabled />)
+  .addStory('Disabled with value: outline', () => <Input disabled value="Disabled" />)
+  .addStory('Disabled with value: underline', () => <Input disabled appearance="underline" value="Disabled" />)
+  .addStory('Disabled with value: filled-darker', () => <Input disabled appearance="filled-darker" value="Disabled" />)
+  .addStory('Disabled with value: filled-lighter', () => (
```

Comment:
```suggestion
  .addStory('Disabled with appearance: outline', () => <Input disabled value="Disabled" />)
  .addStory('Disabled with appearance: underline', () => <Input disabled appearance="underline" value="Disabled" />)
  .addStory('Disabled with appearance: filled-darker', () => <Input disabled appearance="filled-darker" value="Disabled" />)
  .addStory('Disabled with appearance: filled-lighter', () => (
```

Reply:
Renamed to "Disabled, appearance: outline". And for consistency, I've renamed the invalid ones above to "Invalid, appearance: outline".

I'm also going to remove the "Disabled" one (without a value), since it's otherwise the same as the new "Disabled, appearance: outline".